kebechet/managers/thoth_advise/README.rst

```diff
@@ -0,0 +1,58 @@
+Kebechet Advise-Results Manager
+-----------------------
+This manager will manage your Python dependencies using Thoths recommendation system. Your repository will be automatically updated with the optimal Python packages for your project without you having to lift your finger.
+
+A prerequisite for this manager is to have Pipfile and .thoth.yaml, present in the repo. Pipfile should state all direct dependencies (with possible required specifications). .thoth.yaml should contain a valid configuration for thamos
+
+    Pipfile - respecting _`pipenv <https://github.com/pypa/pipenv>`_ tool
+    Pipfile.lock - automatically managed by this manager - states all pinned down versions of your application stack
+
+If you do not have Pipfile.lock present in your repository, this manager will automatically open a pull request with initial dependency lock.
+
+Custom PyPI indexes are supported respecting Pipfile syntax.
```

Comment:
Please link to https://pipenv.readthedocs.io/en/latest/advanced/#specifying-package-indexes